docs/Node_Providers.md

```diff
@@ -0,0 +1,36 @@
+---
+id: Node_Providers
+title: Alternatives
+sidebar_label: Node Providers
+---
+
+To interct with Arbitrum One and/or Arbirum public testnet, you can also rely on popular node providers that offer access to the nodes of these two networks. All you need to do is to choose the provider and the required node, and the node service provider will carry out out all the settings and installation. Here, we list those node services that help you build on Arbitrum One and/or Rinkbey testnet.
+
+## Alchemy
+
+Alchemy provides endpoints for both Arbitrum One (Mainnet) and Rinkeby testnet. Depending one which network you want to interact with, you can use between the following endpoints and interact with directly in the command line or point your MetaMask RPC endpoints to them.
+    
+    Mainnet: https://arb-mainnet.g.alchemy.com/v2/your-api-key
+    Rinkeby: https://arb-rinkeby.g.alchemy.com/v2/your-api-key
+
+*Not sure how to create an Alchemy API key?*
+To use Alchemy's products, you need an API key to authenticate your requests. For dteilaed steps of how to create one, see the [Alchemy's Documentation](https://docs.alchemy.com/alchemy/introduction/getting-started#1.create-an-alchemy-key).
+
+
+## Infura
+
+Infura is a node service provider that provides access to Arbitrum One (Mainnet) and Rinkeby testnet. To start interacting with either of these networks, you need to set up an Infura account and create a project. Next, you will be given a ***Project ID*** (For detailed steps, see the [Infura's Documentation](https://blog.infura.io/getting-started-with-infura-28e41844cc89/)). You can now select which endpoint you want to connect to, this could be Arbirum One, or it could be Rinkeby testnet. Depending on which one you choose, your endpoint URL will look like below. You can use these endpoint URLs to start interacting with Arbitrum.
```

Comment:
```suggestion
Infura is a node service provider that provides access to Arbitrum One (Mainnet) and RinkArby testnet. To start interacting with either of these networks, you need to set up an Infura account and create a project. Next, you will be given a ***Project ID*** (For detailed steps, see the [Infura's Documentation](https://blog.infura.io/getting-started-with-infura-28e41844cc89/)). You can now select which endpoint you want to connect to, this could be Arbitrum One, or it could be Rinkeby testnet. Depending on which one you choose, your endpoint URL will look like below. You can use these endpoint URLs to start interacting with Arbitrum.
```